### ISSUE TITLE
documentation doesn't explain that `termwidth` requires a feature

### ISSUE THREAD

**Issue by Lokathor** [mgeisler/textwrap#165]
It should be made much more clear that [termwidth](https://docs.rs/textwrap/0.11.0/textwrap/fn.termwidth.html) is feature flagged.

**Comment by mgeisler** [comment]
Right, I see what you mean — when looking at the function alone, there's no warning about the feature flag. I'll update the docstring, thanks!

**Comment by rivy** [comment]
It's even more confusing because the currently published version (v0.11.0) uses "term_size", but the repo code *Cargo.toml* has "terminal_size". Neither of which is in any published documentation.
😕

**Comment by mgeisler** [comment]
Hi @rivy ! Ups, you have a great point. It seems I painted myself into a bit of a corner when I replaced `term_size` with `terminal_size` in #181...

I had not considered that people would go and look at `Cargo.toml` in the repository — but it's of course the natural place to examine the source code, including the optional features declared in the `Cargo.toml` file.

I guess we could re-introduce the `term_size` feature as an alias for `terminal_size`. In addition, the docstring needs to be updated to make it clear that the function only exists when a certain feature is enabled.
